### ISSUE TITLE
Document changes to Octicons

### ISSUE THREAD

**Issue by misolori** [Microsoft/vscode-docs#2315]
Once the new version of Octicons is merged into master (https://github.com/Microsoft/vscode/pull/65989) we're going to need proper documentation of these changes in the release notes.

We should also try to address #1094 and provide proper documentation for Octicons in our extensible docs. @octref, do you have any ideas of where this should be placed in our docs?

**Comment by misolori** [comment]
> Are there other places where extension authors could use Octicon?

As far as I know it's only the Status Bar and Quick Pick (see GitLens > Show Commit Details). @bpasero, can you confirm?

**Comment by octref** [comment]
Reminder: need to reference to it in https://github.com/Microsoft/vscode-extension-samples/tree/master/quickinput-sample too, then.

**Comment by bpasero** [comment]
@misolori I think octicons can also be used in hovers, I believe the GitLens extension (@eamodio) is doing that. He might also know more locations from his usage.

**Comment by bpasero** [comment]
Ah ok 👍

**Comment by gregvanl** [comment]
https://code.visualstudio.com/api/references/icons-in-labels
